SPECS/autoconf213/autoconf213.spec

```diff
@@ -62,99 +67,135 @@ sed -i 's/AC_DEFUN(AC_PROG_F77,/AU_DEFUN(AC_PROG_F77,/g' ./acspecific.m4
 sed -i 's/AC_DEFUN(AC_PROG_F77_WORKS,/AU_DEFUN(AC_PROG_F77_WORKS,/g' ./acspecific.m4
 sed -i 's/AC_DEFUN(AC_F77_LIBRARY_LDFLAGS,/AU_DEFUN(AC_F77_LIBRARY_LDFLAGS,/g' ./acspecific.m4
 make check
- 
+
 %post
 /sbin/install-info %{_infodir}/%{name}.info %{_infodir}/dir || :
- 
+
 %preun
 if [ "$1" = 0 ]; then
     /sbin/install-info --delete %{_infodir}/%{name}.info %{_infodir}/dir || :
 fi
- 
-%clean
-rm -rf ${RPM_BUILD_ROOT}
- 
+
+%{clean}
```

Comment:
Sorry, quick nit, since you're here- would you mind removing the `%clean` section here? It's a nice small bit of clean-up that isn't caught by the linter yet. This section is unnecessary in our build system.